pkg/controller/kubelet-config/helpers.go

```diff
@@ -131,6 +132,40 @@ func createNewDefaultFeatureGate() *osev1.FeatureGate {
 	}
 }
 
+func createNewDefaultNodeconfig() *osev1.Node {
+	return &osev1.Node{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: ctrlcommon.ClusterNodeInstanceName,
+		},
+		Spec: osev1.NodeSpec{
+			CgroupMode:           osev1.CgroupModeDefault,
+			WorkerLatencyProfile: osev1.DefaultUpdateDefaultReaction,
+		},
+	}
+}
+
+// updateOriginalKubeConfigwithNodeConfig updates the original Kubelet Configuration based on the Nodespecific configuration
+func updateOriginalKubeConfigwithNodeConfig(node *osev1.Node, originalKubeletConfig *kubeletconfigv1beta1.KubeletConfiguration) error {
```

Comment:
is osev1.Node a node config? If so can we name the variables indicating that? It's a little confusing reading through..

Reply:
Addressed